packages/sui-test/README.md

```diff
@@ -132,6 +132,69 @@ Cypress can be detected as a robot if your server has that kind of protection or
 
 If defined, any error on your tests will create a screenshot of that moment in the `./.tmp/test-e2e/screenshots` folder of your project.
 
+
+# Tools
+
+## Descriptor by environment patcher
+The descriptor by environment is a patch with the purpose of add some extra functionality to our mocha describe and it methods.
+
+### How to import it?
+
+First of all, the patcher MUST BE APPLIED on each test that we want to have the extra methods so at the top of `ourExampleSpec.js` we will add the next code:
+
+```javascript
+import { descriptorsByEnvironmentPatcher } from '@s-ui/test/lib/descriptor-environment-patcher'
+descriptorsByEnvironmentPatcher()
+```
+And that's it, from that line you will have the next methods added to the base of the mocha lib:
+- describe.onlyClient
```

Comment:
I've some concerns about the `onlyClient` naming as it collides with the `only` meaning on the mocha world. Maybe a `describe.forClient` could be a better naming?

Reply:
I'm not the best naming things. :D Alternative: `describe.runOnClient`. Or if you find another more suitable go ahead. :D

Reply:
We have been talking with carlos yesterday about that and we haven't found any alternative so you provide two really good candidates to dismiss the naming confusion. I prefeer describe.runOnClient or describe.runJustOnClient

what do you think?